vhdbuilder/scripts/automate_helpers.sh

```diff
@@ -1,6 +1,20 @@
 #!/bin/bash
 set -x
 
+retrycmd_if_failure() {
+    retries=$1; wait_sleep=$2; shift && shift
+    for i in $(seq 1 $retries); do
+        "${@}" && break || \
+        if [ $i -eq $retries ]; then
+            echo Executed \"$@\" $i times;
```

Comment:
maybe augment the error line with `ERROR` or `FATAL` to maybe it stand out and allow for easier grep. 

We should also print the expected count: `${times}/{$retries}`

Reply:
Or `ERROR: Exhausted all retries (${times}/{$retries}), forcing a failure`

Reply:
I just copy-pasta'd what we already have in our CSE scripts, will like into enhancing

Reply:
fixed